app/frontend/actions.ts

```diff
@@ -1347,54 +1333,32 @@ export default ({setState, getState}: {setState: SetStateFn; getState: GetStateF
         throw NamedError('PoolRegNoHwWallet')
       }
 
+      const {plan, ttl, validityIntervalStart} = state.poolRegTransactionSummary
+
       const txAux = await wallet.getAccount(state.sourceAccountIndex).prepareTxAux(
-        state.poolCertTxVars.plan, // @ts-ignore (Fix byron-shelley formats later)
-        parseInt(state.poolCertTxVars.ttl, 10)
+        plan,
+        // TODO: we should do parsing somewhere else
+        parseCliTtl(ttl),
```

Comment:
doesn't `state.poolRegTransactionSummary` contain already properly parsed ttl? If not, why?